.github/workflows/build_cmake.yml

```diff
@@ -23,20 +22,13 @@ defaults:
 
 jobs:
   build:
-
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-latest', 'windows-latest', 'macos-latest']
+        os: ['ubuntu-latest']
```

Comment:
I fully support this. I don't know what compilers they were using, but last time I checked msvc had barely no support for C++20.